direct/direct.go

```diff
@@ -122,13 +127,14 @@ func NewClient(ctx context.Context, keytype string, mnemonics string, relayURL s
 	}
 
 	cl := &DirectClient{
-		source:    &source,
-		signer:    identity,
-		responses: make(map[string]chan *types.Envelope),
-		twinDB:    twinDB,
-		privKey:   privKey,
-		reader:    reader,
-		writer:    writer,
+		source:           &source,
+		signer:           identity,
+		responses:        make(map[string]chan *types.Envelope),
+		twinDB:           twinDB,
+		privKey:          privKey,
+		enableEncryption: enableEncryption,
```

Comment:
you don't need this flag, your code can just leave the privKey to nill if enableEncryption is disabled.